qiskit/test/testing_options.py

```diff
@@ -1,93 +0,0 @@
-# This code is part of Qiskit.
-#
-# (C) Copyright IBM 2017, 2018.
-#
-# This code is licensed under the Apache License, Version 2.0. You may
-# obtain a copy of this license in the LICENSE.txt file in the root directory
-# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
-#
-# Any modifications or derivative works of this code must retain this
-# copyright notice, and modified files need to carry a notice indicating
-# that they have been altered from the originals.
-
-"""Obtain and set the options in QISKIT_TESTS, used for running the tests."""
-
-import os
-import logging
-
-logger = logging.getLogger(__name__)
-
-
-def get_test_options(option_var="QISKIT_TESTS"):
-    """Read option_var from env and returns a dict in which the test options are set.
-
-    Args:
-        option_var (str): The env var to read. Default: 'QISKIT_TESTS'
-
-    Returns:
-        dict: A dictionary with the format {<option>: (bool)<activated>}.
-    """
-    tests_options = {"skip_online": False, "mock_online": False, "run_slow": False, "rec": False}
```

Comment:
Only `run_slow` is still in use, and I inlined the one use of that to the `slow_test` decorator.